src/main/java/org/generationcp/middleware/dao/oms/CvTermPropertyDao.java

```diff
@@ -87,6 +86,18 @@ public List getByCvId(Integer cvId) throws MiddlewareQueryException {
 		return properties;
 	}
 
+	public List<CVTermProperty> getByCvTermIdsAndType(final List<Integer> cvTermIds, final int typeId) throws MiddlewareQueryException {
```

Comment:
Remove throwing of MiddlewareQueryException (all methods) and perform code cleanup in this class